docs/custom_build.md

```diff
@@ -20,9 +34,56 @@ custom_build(
 )
 ```
 
-One piece of this needs more explanation: `$EXPECTED_REF`.
+Tilt will run this command to build the image, verify that the image is in Docker,
```

Comment:
> verify that the image is in Docker
imo a little clearly to say something like: "verify that the expected image exists in your local Docker image registry, and (if necessary) push the image..."

(Idk if "local Docker image registry" is the right verbiage but you get the idea)

Reply:
done